lib/issues.js

```diff
@@ -108,39 +120,46 @@ const IMPACT_LABEL_RE = /(?:\*\*)?Impact(?:\*\*)?:(?:\*\*)? ([^\n]+)?/
 const CHANCE_LABEL_RE = /(?:\*\*)?Likelihood(?:\*\*)?:(?:\*\*)? ([^\n]+)?/
 
 exports.parseNotionMarkdownIssues = function parseNotionMarkdownIssues(markdown) {
-  return markdown.split(/(?=##\s+\([CHMLQGI]-\d\))/)
+  return markdown.split(new RegExp(`(?=${TITLE_LINE_RE})`))
```

Comment:
I wasn't able to get the Issue Formatter to work with pasting in multiple issues at once: either before or after these changes.  Not sure if this was working yet?  I've just been pasting them in one-at-a-time for now.